src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -224,6 +230,26 @@ export const AdministrationPage = () => {
           </ServiceContainerCreatingUpdating>
         </FlexItem>
 
+        <FlexItem>
+          <ContentBox padding title="Git version">
+            <ServiceContainerLoading {...serviceContainerCurrentPncWebUiCommit} title="PNC Web UI GitHub Commit">
+              <Attributes>
+                <AttributesItem title="GitHub main deployed">
+                  {serviceContainerCurrentPncWebUiCommit.data?.sha.startsWith(process.env.REACT_APP_GIT_SHORT_SHA!)
+                    ? 'yes'
+                    : 'no'}
+                </AttributesItem>
+
+                <AttributesItem title="Deployed commit SHA">{process.env.REACT_APP_GIT_SHORT_SHA}</AttributesItem>
+
+                <AttributesItem title="GitHub main commit">
```

Comment:
I would go with "Latest commit SHA"

Reply:
On PM we discussed inline loading could be preferred not to block "Deployed commit SHA" when request fails due to limits. First one item (yes/no) would not be displayed at all.